sim_city.py

```diff
@@ -0,0 +1,69 @@
+class City:
+    def __init__(self, base_city_tax=1000):
+        self.neighborhoods = {}
+        self.base_city_tax = base_city_tax
+
+    def how_much_money(self):
+        city_tax = self.base_city_tax
+        for neighborhood in self.neighborhoods:
+            city_tax += self.neighborhoods[neighborhood].how_much_money_neighborhood()
+        return city_tax
+
+    def build_a_neighborhood(self, neighborhood_name):
+        new_neighborhood = Neighborhood()
+        self.neighborhoods[neighborhood_name] = new_neighborhood
+        self.base_city_tax *= 1.1
+
+    def ruin_a_neighborhood(self, neighborhood_name):
+        if neighborhood_name not in self.neighborhoods:
+            return
+        del self.neighborhoods[neighborhood_name]

```

Comment:
Better to use `pop`, since `del` can behave unexpectedly in certain occasions.

Reply:
Done